extra/language/rust.nix

```diff
@@ -30,8 +46,32 @@ with lib;
       # Used by tools like rust-analyzer
       name = "RUST_SRC_PATH";
       value = toString cfg.packageSet.rustPlatform.rustLibSrc;
-    }];
+    }]
+    ++
+    (lib.optionals hasLibraries [
+      {
+        name = "LD_LIBRARY_PATH";
+        prefix = "$DEVSHELL_DIR/lib";
+      }
+    ])
+    ++ lib.optionals hasIncludes [
+      {
+        name = "LD_INCLUDE_PATH";
```

Comment:
Never heard of an LD_INCLUDE_PATH. I think you are missing things up here.
There is an INCLUDE_PATH for header files and there is an LD_LIBRARY_PATH to find shared libraries at runtime.
And there is LIBRARY_PATH for linking.

Reply:
I'm guessing this is resolved since #120 was closed.

Reply:
Not really, as the unused envvar is still present. #120 was closed *because* LD_INCLUDE_PATH was renamed to C_INCLUDE_PATH, which was the correct name.